From d34d825473a50a66b9b21d0a5b072fc12f71821c Mon Sep 17 00:00:00 2001
From: Christina Fu <cfu@redhat.com>
Date: Thu, 22 Mar 2018 11:39:05 -0700
Subject: [PATCH 1/3] fix TPS CS.cfg param from tps.connector.connCAList to
 tps.connCAList

Change-Id: Ic391b845358736daab4b814c86e6f7f512a209bb
(cherry picked from commit 4e237910d02918a53618154bf923204b3e2be085)
(cherry picked from commit 3f0f0d971e4382ef9ea1490d1a07644fcef0b343)
---
 base/tps/man/man5/pki-tps-connector.5 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/tps/man/man5/pki-tps-connector.5 b/base/tps/man/man5/pki-tps-connector.5
index b3e405e..a2df454 100644
--- a/base/tps/man/man5/pki-tps-connector.5
+++ b/base/tps/man/man5/pki-tps-connector.5
@@ -68,7 +68,7 @@ Example ops: enrollment, renewal, revoke, unrevoke, getcert.
 
 This property contains the connection timeout.
 
-.SS tps.connector.connCAList
+.SS tps.connCAList
 
 This property is used for \fIRevocation Routing\fP. It contains a list of ordered ca id's separated by ',' that the revocation attempt should be made to.
 Example:
-- 
1.8.3.1


From fbd5122bd3e7ffe3632406f006630e2ddbc41907 Mon Sep 17 00:00:00 2001
From: Christina Fu <cfu@redhat.com>
Date: Mon, 26 Mar 2018 10:09:42 -0700
Subject: [PATCH 2/3] reflect dogtagpki url change in CMCRequest man page.

Change-Id: I8eb5884a26850b87f378c4417939c873c27fd409
(cherry picked from commit 3ce3ae9bd73a906beb0254a7488b15e11ddcf905)
(cherry picked from commit 08c6684dfbabb53ab9301e33e2b40389b93f18bc)
---
 base/java-tools/man/man1/CMCRequest.1 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/java-tools/man/man1/CMCRequest.1 b/base/java-tools/man/man1/CMCRequest.1
index f647ef7..8c67fc0 100644
--- a/base/java-tools/man/man1/CMCRequest.1
+++ b/base/java-tools/man/man1/CMCRequest.1
@@ -174,7 +174,7 @@ For revocation by authenticating with user's shared secret, the following parame
 CMC requests must be submitted to the CA to be processed. Tool supported by Dogtag for submitting these requests is \fBHttpClient\fP.
 .PP
 \fBNote:\fP For examples on how to use this tool, please see
-http://pki.fedoraproject.org/wiki/PKI_10.4_CMC_Feature_Update_(RFC5272)#Practical_Usage_Scenarios for Practical Usage Scenarios, and their examples.
+http://www.dogtagpki.org/wiki/PKI_10.4_CMC_Feature_Update_(RFC5272)#Practical_Usage_Scenarios for Practical Usage Scenarios, and their examples.
 
 .SH AUTHORS
 Christina Fu <cfu@redhat.com>.
-- 
1.8.3.1


From 7b5a899e3d237e2be3bc1c7d9e4dd7613cdd9f24 Mon Sep 17 00:00:00 2001
From: Christina Fu <cfu@redhat.com>
Date: Thu, 29 Mar 2018 09:59:02 -0700
Subject: [PATCH 3/3] quick fix on wrong keyType in profile

Change-Id: I0fa90ebb559e0fb8af123191f7bc7cdedbc55d87
(cherry picked from commit 995682153e10393dc46f16090c26f28ca1b6cfc6)
(cherry picked from commit 3e0f7dfac7c8ba0859aad6a082eb1659631e2620)
---
 base/ca/shared/profiles/ca/caUserCert.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base/ca/shared/profiles/ca/caUserCert.cfg b/base/ca/shared/profiles/ca/caUserCert.cfg
index 1f62bb8..9164dac 100644
--- a/base/ca/shared/profiles/ca/caUserCert.cfg
+++ b/base/ca/shared/profiles/ca/caUserCert.cfg
@@ -36,7 +36,7 @@ policyset.userCertSet.2.default.params.range=180
 policyset.userCertSet.2.default.params.startTime=0
 policyset.userCertSet.3.constraint.class_id=keyConstraintImpl
 policyset.userCertSet.3.constraint.name=Key Constraint
-policyset.userCertSet.3.constraint.params.keyType=EC
+policyset.userCertSet.3.constraint.params.keyType=RSA
 policyset.userCertSet.3.constraint.params.keyParameters=1024,2048,3072,4096
 policyset.userCertSet.3.default.class_id=userKeyDefaultImpl
 policyset.userCertSet.3.default.name=Key Default
-- 
1.8.3.1

